src/core/transaction.js

```diff
@@ -1186,9 +1162,6 @@ Transaction.prototype.pathAdd = function(path) {
   if (!Array.isArray(path)) {
     throw new Error('Path must be an array');
   }
-  // if (this.getType() !== 'Payment') {
```

Comment:
Eventually we'll subclass Transaction so they'd be unnecessary... I doubt that we'll ever switch these on prior to that, so just removing some clutter